packages/ui-extensions/docs/surfaces/point-of-sale/staticPages/pages/versions.doc.ts

```diff
@@ -17,7 +17,98 @@ const data: LandingTemplateSchema = {
       type: 'Generic',
       anchorLink: 'compatibility-policy',
       title: 'Compatibility Policy',
-      sectionContent: '',
+      sectionContent: `
+# POS UI Extensions Versioning
+
+This guide explains how versioning works for POS UI Extensions, including how to manage API versions, understand release schedules, and ensure merchants can access your extensions at the right time.
+
+## Understanding POS UI Extensions Versioning
+
+POS UI Extensions follow Shopify's quarterly API versioning system while also depending on the Shopify POS app release schedule. You need to understand both aspects to plan your development and release strategy effectively.
+
+### API Versioning
+- Shopify releases POS UI Extensions versions quarterly (January, April, July, October) following the [Shopify API versioning schedule](https://shopify.dev/docs/api/usage/versioning)
```

Comment:
Redundant and _if_ the schedule ever changes months:

```suggestion
- Shopify releases POS UI Extensions versions quarterly following the [Shopify API versioning schedule](https://shopify.dev/docs/api/usage/versioning)
```